### ISSUE TITLE
Trying to launch MulticMC with Fabric for 1.14.4

### ISSUE THREAD

**Issue by ah-callie** [MultiMC/MultiMC5#2765]
System Information
-----------------------------
MultiMC version: 0.6.6-1341

Operating System: 64bit

Summary of the issue or suggestion: Instant crash when trying to launch
----------------------------------------------

What should happen:
------------------------------

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.

2.

3.

Suspected cause:
---------------------------
I'm not sure why when I update my Java it's still giving me a 32 bit version (I think?) when I should have a 64 bit version, I think this is the cause.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
https://paste.ee/p/fnFEj

Additional Info:
---------------------------

**Comment by S10MC2015** [comment]
According to people in Fabric Discord, You are missing Fabric API which is required for VoxelMap.

**Comment by calloatti** [comment]
You are missing https://www.curseforge.com/minecraft/mc-mods/fabric-api

The Fabric loader cannot report it correctly since voxelmap is not correctly listing the dependency.

Also the file in curseforge does not list fabric-api as a required dependency.

There is a tutorial here that you must not know about, otherwise you would have already had the fabric api mod installed: https://fabricmc.net/wiki/tutorial:install_with_multimc

So basically not a MultiMC issue.
